Source/WSTagsField.swift

```diff
@@ -276,8 +276,9 @@ public class WSTagsField: UIView {
     }
     
     public func acceptCurrentTextAsTag(){
-        self.addTag(self.textField.text!)
-        tokenizeTextFieldText()
+        if self.textField.text != ""{
```

Comment:
Ah. Better idea. Swift is still very new to me. Thanks for the excellent write up.